src/erlcloud_access_analyzer.erl

```diff
@@ -0,0 +1,244 @@
+    delete_analyser(AwsConfig, AnalyzerName, _Params = []).
+
+-spec delete_analyser(AwsConfig, AnalyzerName, Params) -> Result
+when AwsConfig :: aws_config(),
+     AnalyzerName :: binary() | string(),
+     Params :: params(),
+     Result :: ok | {error, not_found} | {error, term()}.
+delete_analyser(AwsConfig, AnalyzerName, Params) ->
+    Path = ["analyzer", AnalyzerName],
+    case request(AwsConfig, _Method = delete, Path, Params) of
+        ok ->
+            ok;
+        {error, {<<"ResourceNotFoundException">>, _Message}} ->
+            {error, not_found};
+        {error, Reason} ->
+            {error, Reason}
+    end.
+
+%% -----------------------------------------------------------------------------
+%% Local functions
+%% -----------------------------------------------------------------------------
+
+list_analyzers_next(AwsConfig, Params, Token0, Analyzers0) ->
+    case list_analyzers(AwsConfig, [{<<"nextToken">>, Token0} | Params]) of
+        {ok, Analyzers} ->
+            Analyzers1 = lists:append(Analyzers0, Analyzers),
+            {ok, Analyzers1};
+        {ok, Analyzers, Token1} ->
+            Analyzers1 = lists:append(Analyzers0, Analyzers),
+            list_analyzers_next(AwsConfig, Params, Token1, Analyzers1);
+        {error, Reason} ->
+            {error, Reason}
+    end.
+
+request(AwsConfig, Method, Path) ->
+    request(AwsConfig, Method, Path, _Params = []).
+
+request(AwsConfig, Method, Path, Params) ->
+    request(AwsConfig, Method, Path, Params, _RequestBody = <<>>).
+
+request(AwsConfig0, Method, Path, Params, RequestBody) ->
+    case erlcloud_aws:update_config(AwsConfig0) of
+        {ok, AwsConfig1} ->
+            AwsRequest0 = init_request(AwsConfig1, Method, Path, Params, RequestBody),
+            AwsRequest1 = erlcloud_retry:request(AwsConfig1, AwsRequest0, fun should_retry/1),
+            case AwsRequest1#aws_request.response_type of
+                ok ->
+                    decode_response(AwsRequest1);
+                error ->
+                    decode_error(AwsRequest1)
+            end;
+        {error, Reason} ->
+            {error, Reason}
+    end.
+
+init_request(AwsConfig, Method, Path, Params, Payload) ->
+    Host = AwsConfig#aws_config.access_analyzer_host,
+    Service = "access-analyzer",
+    NormPath = norm_path(Path),
+    NormParams = norm_params(Params),
+    Region = erlcloud_aws:aws_region_from_host(Host),
+    Headers = [{"host", Host}, {"content-type", "application/json"}],
+    SignedHeaders = erlcloud_aws:sign_v4(Method, NormPath, AwsConfig, Headers, Payload, Region, Service, Params),
+    #aws_request{
+        service = access_analyzer,
+        method = Method,
+        uri = "https://" ++ Host ++ NormPath ++ NormParams,
+        request_headers = SignedHeaders,
+        request_body = Payload
+    }.
+
+norm_path(Path) ->
+    binary_to_list(iolist_to_binary(["/" | lists:join("/", Path)])).
+
+norm_params([] = _Params) ->
+    "";
+norm_params(Params) ->
```

Comment:
looks like we need to export this [function](https://github.com/erlcloud/erlcloud/blob/master/src/erlcloud_aws.erl#L1197) and use it here instead of `norm_params()`

Reply:
exported